tasks/pythonz.yml

```diff
@@ -4,6 +4,7 @@
     pkg: "{{ item }}"
     state: latest
     force: yes
+    update_cache: yes
```

Comment:
In most our usages in the pyslackers group we depend on [pyslackers/ansible-role-common)(https://github.com/pyslackers/ansible-role-common/blob/6f9cff963bd8f3b1bc85766c6134bb8abd7080b9/tasks/common.yml) to do this, but seems like a reasonable thing to change. The only caveat being to use the `cache_valid_time: "{{ apt_cache_valid_time }}"` we use there.  Thoughts @ovv ?

Reply:
I would prefer not needing to add the `apt_cache_valid_time` variable to this playbook as it's not really related to python.
I'm not really sure what the errors was but ideally this step should be done in another role.

Reply:
For the bug itself, it arrives when the Debian repository has a new package release and the old package is removed: The local Debian cache still has an old reference to the previous package version.
After thinking, it might be good that Ansible has an option for apt to avoid to do several update_cache in the same playbook session: Easier to share between roles than a custom parameter that it would be different between roles.